src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml

```diff
@@ -588,7 +588,7 @@
                                        FontSize="16"
                                        ToolTip="{x:Static props:Resources.Distance_adjacent_zones}"
                                         AutomationProperties.Name="{x:Static props:Resources.Distance_adjacent_zones}"
-                                       Text="&#xF617;" />
+                                       Text="&#xE944;" />
```

Comment:
"E944" is also called "ReturnToWindow". Not sure if that's the best icon for this. Maybe one of the "PageMargin**" icons on https://docs.microsoft.com/nl-nl/windows/apps/design/style/segoe-fluent-icons-font#pua-f600-f800 ? Or maybe "Trim" (`e78a`)?

Reply:
At that time I was just searching for a nice looking icon..

Reply:
This is Trim
![image](https://user-images.githubusercontent.com/36418285/173053491-f23e111c-8af7-4ccd-b8ac-7b17d43795c2.png)

Page Margin, too similar with the second icon?
![image](https://user-images.githubusercontent.com/36418285/173053535-866a2150-2340-4fb5-9018-aa902576912f.png)

FitPage
![image](https://user-images.githubusercontent.com/36418285/173053719-4bc91eb7-deeb-4a63-ab44-8d48e89ef59d.png)

Reply:
Maybe trim looks better?